libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx

```diff
@@ -58,11 +57,10 @@ export const TemplatesDesigner = ({
       connectionsError ||
       Object.values(parametersError)?.filter((error) => error).length > 0
     ) {
-      // TODO: Show error message
-      console.log('Error checking conditions before calling createWorkflowCall');
-      return;
+      throw new Error('Missing information for workflow creation');
```

Comment:
not blocking, but we need to see if this info gets displayed in UI if yes, then we should localise it